src/Tests/CaptchaTestCase.php

```diff
@@ -187,9 +188,5 @@ public function testCaptchaOnLoginBlockOnAdminPagesIssue893810() {
     // @todo This assumes standard profile login form block, manually add it?
     $this->drupalGet('');
     $this->assertCaptchaPresence(TRUE);
-
-    // Check there is a CAPTCHA on "forbidden" admin pages.
-    $this->drupalGet('admin');
```

Comment:
again, just removing the test isn't really correct, we need to figure out what it was trying to test.